pkg/command/run.go

```diff
@@ -135,3 +130,10 @@ func (m *manager) StoreCommandLogs(ctx context.Context, blobPath string, reader
 	}()
 	return errChan
 }
+
+func (m *manager) closeAndWriteLog(azureWriter *io.PipeWriter, errChan <-chan error, commandType core.CommandType) {
```

Comment:
Do we get partial logs when pre-run command failed ?
or we get error msg only ?

Reply:
we aren't getting any logs in case pre-run command fails